askama_derive/src/input.rs

```diff
@@ -127,13 +132,16 @@ impl TemplateInput<'_> {
             let mut nested = vec![parsed.nodes()];
             while let Some(nodes) = nested.pop() {
                 for n in nodes {
-                    let mut add_to_check = |path: Rc<Path>| -> Result<(), CompileError> {
-                        if !map.contains_key(&path) {
+                    let mut add_to_check = |new_path: Rc<Path>| -> Result<(), CompileError> {
```

Comment:
Please avoid/isolate unrelated changes.

Reply:
It's not unrelated, I use the captured parent `path`, but to do so I needed to rename the closure argument. I use it here:

```rust
                            let source = get_template_source(
                                &new_path,
                                Some((&path, parsed.source(), n.template_content())),
                            )?;
```